modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/service/SchedulerChannelProvider.java

```diff
@@ -183,19 +181,12 @@ private String buildGetTriggerQuery(String subject) {
 
         StringBuilder sb = new StringBuilder();
 
-        sb.append("SELECT ");
-        sb.append(getCorrectName(JOB_NAME, isPostgres));
-        sb.append(", ");
-        sb.append(getCorrectName(JOB_DESCRIPTION, isPostgres));
-        sb.append(", ");
-        sb.append(getCorrectName(JOB_DATA, isPostgres));
-        sb.append(" FROM ");
-        sb.append(getCorrectName(QRTZ_JOB_DETAILS, isPostgres));
-        sb.append(" WHERE ");
-        sb.append(getCorrectName(JOB_NAME, isPostgres));
-        sb.append(" = \'");
-        sb.append(subject);
-        sb.append("\'");
+        sb.append("SELECT ").append(getCorrectName(JOB_NAME, isPostgres)).append(", ")
```

Comment:
to be honest the previous version looked more clear, maybe a matter of preference